server/base/src/main/java/org/apache/accumulo/server/AbstractServer.java

```diff
@@ -181,6 +190,9 @@ public void startServiceLockVerificationThread() {
             }
           });
       verificationThread.start();
+    } else {
+      log.debug(
```

Comment:
Could make this info as it will only run once  also the prop name would be useful, like `"+GENERAL_SERVER_LOCK_VERIFICATION_INTERVAL.getKey(+")`

Reply:
Fixed in c56c870